chia/rpc/rpc_server.py

```diff
@@ -302,40 +302,54 @@ async def start_rpc_server(
     Starts an HTTP server with the following RPC methods, to be used by local clients to
     query the node.
     """
```

Comment:
this refactor seems unrelated to the data-layer, i suggest you cherry pick it into another pr

Reply:
(discussed elsewhere, for the record here...)  It certainly is, it was added to identify a problem I ran into here so I kept it around.  But yes, if committed to chia-blockchain it would be via a different PR for just this.